department-of-veterans-affairs/va.gov-cms
# VACMS_3616: Add html table field html removal script.

## Description
See #3616 

## Testing done
`lando drush php-script VACMS-3616-remove-tablefield-tags-2020-12.php --script-path=/app/scripts/content`

## QA steps
Run the above script, and spot check the hours fields on the following pages:
- node/603/edit
- node/605/edit
- node/1023/edit
- node/478/edit

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
